content/800-data-platform/250-environments/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Environments'
+metaTitle: 'Environments'
+metaDescription: 'In a project in the Prisma Data Platform, you can create multiple environments to associate a Prisma schema on a repository branch with a database and, thus, separate database development activities with JavaScript/TypeScript and Prisma ORM.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+For any project in the Prisma Data Platform, you can create multiple environments and associate each environment with a different database, Data Proxy location, and a different branch of your GitHub repository.
+
+</TopBlock>
+
+## About environments in the Prisma Data Platform
+
+You can dedicate each environment to a different stage of your database application development, such as production, staging, development, or any others that make sense for your project.
```

Comment:
```suggestion
You can dedicate each environment to a different stage of your database application life-cycle, such as production, staging, development, or any others that make sense for your project.
```

Or maybe production cycle? Or "development process"? Or something else? Just to avoid "development" as a single word twice in slightly different contexts (because as written, we have "development" as part of "development", which is slightly confusing).